numba/tests/test_parfors_passes.py

```diff
@@ -66,6 +69,8 @@ def _run_parfor(cls, test_func, args):
                 tp.state.typingctx, tp.state.func_ir, tp.state.args, None
             )
 
+            typed_passes.PreLowerStripPhis().run_pass(tp.state)
```

Comment:
hmmm, is there any reason for this to be typed other than it is used post type inference?

Reply:
The generator handling uses `typemap`